ui/hooks/snaps/useTransactionInsightSnaps.js

```diff
@@ -98,16 +84,12 @@ export function useTransactionInsightSnaps({
     };
   }, [
     transaction,
-    eagerFetching,
     chainId,
     origin,
     subjects,
     // TODO: Figure out how to improve this
     JSON.stringify(insightSnaps),
-    insightSnapId,
-    ///: BEGIN:ONLY_INCLUDE_IF(build-flask)
-    hasFetchedV2Insight,
-    ///: END:ONLY_INCLUDE_IF
+    hasFetchedInsight,
   ]);
 
   return { data, loading };
```

Comment:
If we don't combine the two hooks into one, this should at least match the signature insights hook by returning the `warnings`.

Reply:
Yeah, I'll change this.